client/my-sites/backup/backup-contents-page/file-browser/file-browser-context.tsx

```diff
@@ -20,16 +26,17 @@ export const useFileBrowserContext = () => {
 interface FileBrowserProviderProps {
 	children: React.ReactNode;
 	locale: string;
+	notices: NoticeHandlers;
 }
 
 /**
  * Provider that wraps backup pages with FileBrowser context
  */
-export function FileBrowserProvider( { children, locale }: FileBrowserProviderProps ) {
+export function FileBrowserProvider( { children, locale, notices }: FileBrowserProviderProps ) {
 	const fileBrowserState = useFileBrowserState();
 
 	return (
-		<FileBrowserContext.Provider value={ { fileBrowserState, locale } }>
+		<FileBrowserContext.Provider value={ { fileBrowserState, locale, notices } }>
```

Comment:
This is handy because it lets us support the notice handler from both Calypso and the Hosting Dashboard.